frida-gum/src/interceptor.rs

```diff
@@ -30,10 +30,7 @@ impl<'a> Interceptor<'a> {
     /// Obtain an Interceptor handle, ensuring that the runtime is properly initialized. This may
     /// be called as many times as needed, and results in a no-op if the Interceptor is
     /// already initialized.
-    pub fn obtain<'b>(_gum: &'b Gum) -> Interceptor
-    where
-        'b: 'a,
-    {
+    pub fn obtain<'b: 'a>(_gum: &'b Gum) -> Interceptor<'b> {
```

Comment:
Why is the `'a` lifetime here?

Reply:
what about this?

Reply:
`'b: 'a` ensures that the lifetime of the input is at least as long as `'a`. I had to put that because https://github.com/frida/frida-rust/commit/117003b97356342ae49c26161225b0354270462b broke the `no_std` test

Reply:
Oh I didn't realize `'a` was on the Interceptor.